shared/chat/conversation/index.desktop.js

```diff
@@ -39,6 +41,7 @@ const Conversation = (props: Props) => {
         onEditMessage={props.onEditMessage}
         onLoadAttachment={props.onLoadAttachment}
         onLoadMoreMessages={props.onLoadMoreMessages}
+        onFocusInput={() => _input && _input.focusInput()}
```

Comment:
ok but can we make this a method and not an inline. let's try and avoid this pattern moving forward if possible